bokehjs/src/lib/models/plots/plot_canvas.ts

```diff
@@ -1190,4 +1217,20 @@ export class PlotView extends LayoutDOMView implements Renderable {
     }
     return {x: NaN, y: NaN}
   }
+
+  protected _messages: Map<string, number> = new Map()
+
+  notify_about(message: string): void {
+    if (this._messages.has(message)) {
+      return
+    }
+    const el = new Div({children: [message]})
+    const timer = setTimeout(() => {
+      this._messages.delete(message)
+      this._notifications.elements = this._notifications.elements.filter((item) => item != el)
+    }, 2000)
+    this._messages.set(message, timer)
+    this._notifications.elements = [...this._notifications.elements, el]
+    logger.info(message)
+  }
```

Comment:
I expect many users would find something like this extremely useful if we can find a good way to expose it

Reply:
I intend to generalize this in future PRs for 3.5. For now this implementation is just enough to make this PR viable.

Reply:
I will probably use issue #6884 to follow up on this.